editor/src/components/canvas/canvas-strategies/absolute-move-strategy.spec.browser2.tsx

```diff
@@ -0,0 +1,373 @@
+            data-uid='ccc'
+          />
+          <div
+            style={{ backgroundColor: '#0091FFAA', position: 'absolute', left: 49, top: 27, width: 200, height: 120 }}
+            data-uid='bbb'
+            data-testid='bbb'
+          />
+        </div>
+      `),
+    )
+  })
+})
+
+describe('Absolute Move Strategy Canvas Controls', () => {
+  it('when an absolute positioned element is started to be moved parent outlines become visible', async () => {
+    const renderResult = await renderTestEditorWithCode(
+      makeTestProjectCodeWithSnippet(`
+        <div style={{ width: '100%', height: '100%' }} data-uid='aaa'>
+          <div
+            style={{ backgroundColor: '#0091FFAA', position: 'absolute', left: 40, top: 50, right: 160, bottom: 230 }}
+            data-uid='bbb'
+            data-testid='bbb'
+          />
+        </div>
+      `),
+      'await-first-dom-report',
+    )
+
+    const parentOutlineControlBeforeDrag =
+      renderResult.renderedDOM.queryByTestId('parent-outlines-control')
+    expect(parentOutlineControlBeforeDrag).toBeNull()
+    const parentBoundsControlBeforeDrag =
+      renderResult.renderedDOM.queryByTestId('parent-bounds-control')
+    expect(parentBoundsControlBeforeDrag).toBeNull()
+
+    const target = EP.appendNewElementPath(TestScenePath, ['aaa', 'bbb'])
+    await renderResult.dispatch([selectComponents([target], false)], true)
+
+    await act(async () => {
+      const dispatchDone = renderResult.getDispatchFollowUpActionsFinished()
+      await renderResult.dispatch(
+        [
+          CanvasActions.createInteractionSession(
+            createInteractionViaMouse(zeroCanvasPoint, emptyModifiers, {
+              type: 'BOUNDING_AREA',
+              target: target,
+            }),
+          ),
+        ],
+        false,
+      )
+      await dispatchDone
+    })
+
+    const parentOutlineControl = renderResult.renderedDOM.getByTestId('parent-outlines-control')
+    expect(parentOutlineControl).toBeDefined()
+    const parentBoundsControl = renderResult.renderedDOM.getByTestId('parent-bounds-control')
+    expect(parentBoundsControl).toBeDefined()
+  })
+  it('when an absolute positioned element is selected the pin lines are visible', async () => {
+    const renderResult = await renderTestEditorWithCode(
+      makeTestProjectCodeWithSnippet(`
+        <div style={{ width: '100%', height: '100%' }} data-uid='aaa'>
+          <div
+            style={{ backgroundColor: '#0091FFAA', position: 'absolute', left: 40, top: 50, right: 160, bottom: 230 }}
+            data-uid='bbb'
+            data-testid='bbb'
+          />
+        </div>
+      `),
+      'await-first-dom-report',
+    )
+
+    await renderResult.dispatch(
+      [selectComponents([EP.appendNewElementPath(TestScenePath, ['aaa', 'bbb'])], false)],
+      true,
+    )
+
+    const pinLineLeft = renderResult.renderedDOM.getByTestId('pin-line-left')
+    expect(pinLineLeft).toBeDefined()
```

Comment:
is it worth checking the actual positions of these lines? is there a robust way to check that, but doesn't break if we change the pin's design?